pactus-wallet history add panic

### What Happened

when run command `pactus-wallet history add` get panic.

```shell
$ pactus-wallet history add

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.buildAddToHistoryCmd.func1(0xc0004e3400?, {0xc0004da640?, 0x4?, 0x105e9bc?})
	github.com/pactus-project/pactus/cmd/wallet/history.go:32 +0x138
github.com/spf13/cobra.(*Command).execute(0xc0004eac00, {0xc0004da600, 0x4, 0x4})
	github.com/spf13/cobra@v1.7.0/command.go:944 +0x863
github.com/spf13/cobra.(*Command).ExecuteC(0xc00037cc00)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
	github.com/pactus-project/pactus/cmd/wallet/main.go:69 +0x21b

```

--- a/cmd/wallet/history.go
+++ b/cmd/wallet/history.go
@@ -25,6 +25,7 @@ func buildAddToHistoryCmd(parentCmd *cobra.Command) {
 	addToHistoryCmd := &cobra.Command{
 		Use:   "add [flags] <ID>",
 		Short: "adds a transaction to the wallet's history.",
+		Args:  cobra.ExactArgs(1),
 	}
 	parentCmd.AddCommand(addToHistoryCmd)
 

